share/instruments/instruments.xml

```diff
@@ -8791,7 +8789,6 @@
                   <genre>common</genre>
                   <genre>orchestra</genre>
             </Instrument>
-            <!-- Electronic Organ has been removed as the term is ambiguous it has been replaced by several different organ types-->
```

Comment:
Comment removed intentionally. It has been recorded in a [separate spreedsheet tab](https://docs.google.com/spreadsheets/d/1SwqZb8lq5rfv5regPSA10drWjUAoi65EuMoYtG-4k5s/edit#gid=1466210644) for now.